cri/v1alpha1/cri.go

```diff
@@ -822,6 +822,19 @@ func (c *CriManager) ContainerStats(ctx context.Context, r *runtime.ContainerSta
 func (c *CriManager) ListContainerStats(ctx context.Context, r *runtime.ListContainerStatsRequest) (*runtime.ListContainerStatsResponse, error) {
 	opts := &mgr.ContainerListOption{All: true}
 	filter := func(c *mgr.Container) bool {
+		if c.Config.Labels[containerTypeLabelKey] != containerTypeLabelContainer {
+			return false
+		}
```

Comment:
Maybe add a blank here to distinguish between filtering created by CRI Manager and filtering by request and it's beeter to write comments. WDTY?

Reply:
Updated